<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>1D Schr√∂dinger Equation Solver</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="CustomChartView" name="chartView" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Initial Wave Packet</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="initialWavePacketComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="sigmaLabel">
        <property name="text">
         <string>Sigma:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="sigmaSpinBox">
        <property name="decimals">
         <number>7</number>
        </property>
        <property name="value">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="x0Label">
        <property name="text">
         <string>x0:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="x0SpinBox">
        <property name="decimals">
         <number>7</number>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="k0Label">
        <property name="text">
         <string>k0:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="k0SpinBox">
        <property name="decimals">
         <number>7</number>
        </property>
        <property name="value">
         <double>50.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Method</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="methodComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Potential</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="potentialComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="sizeLabel">
        <property name="text">
         <string>Simulation Size:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sizeSpinBox">
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dtLabel">
        <property name="text">
         <string>dt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="dtSpinBox">
        <property name="decimals">
         <number>7</number>
        </property>
        <property name="minimum">
         <double>0.000000100000000</double>
        </property>
        <property name="maximum">
         <double>0.100000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000001000000000</double>
        </property>
        <property name="value">
         <double>0.000010000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>dx</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="dxSpinBox">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>0.000010000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
        <property name="value">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pauseResumeButton">
        <property name="text">
         <string>Pause</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveDataButton">
        <property name="text">
         <string>Save Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="probDensityCheckBox">
        <property name="text">
         <string>Show Probability Density</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="realPartCheckBox">
        <property name="text">
         <string>Show Real Part</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="imagPartCheckBox">
        <property name="text">
         <string>Show Imaginary Part</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="potentialCheckBox">
        <property name="text">
         <string>Show Potential</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="autoZoomCheckBox">
        <property name="text">
         <string>Auto Zoom Y-Axis</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="expectationPositionLabel">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Expectation Position &lt;x&gt;:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="expectationMomentumLabel">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Expectation Momentum &lt;p&gt;:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="expectationEnergyLabel">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Expectation Energy &lt;H&gt;:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="timeLabel">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Time: 0.0</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomChartView</class>
   <extends>QWidget</extends>
   <header>CustomChartView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
